### DISCUSSION TITLE
Governing Board Candidates 2024

### DISCUSSION THREAD

**Discussion by wipfli** [maplibre/maplibre#375] [Governing Board Election]
The Governing Board elections will take place online on Thursday, August 29th, 2024. For a full timeline see https://github.com/maplibre/maplibre/issues/374.

Candidates for the Governing Board can register themselves up to two weeks before the election, i.e., until Thursday, August 15th, 2024 by posting in this thread that they are interested in serving on the Board. Please include a description of your involvement in MapLibre and disclose if you are paid for working on MapLibre or have a vested interested in the Project.

Each nomination should be a separate comment.

This discussion should only be used for the candidate messages. As such:
- Please do not upvote or downvote candidates.
- Please do not ask questions below their statements.

If you have questions for the candidates a good place to ask would be in a separate github discussion or in the [MapLibre Slack channel](https://slack.openstreetmap.us/).

**Comment by nyurik** [comment]
Dear MapLibre Community,

I'm seeking your support to continue serving on the MapLibre board of directors. As a passionate advocate for open-source mapping software, I'm committed to driving MapLibre's mission forward.  After the license change, I proposed community to get together under `MapLibre` name, co-founded our organization, set up OpenCollective, did the initial re-branding and CI automation of the code.

### My Accomplishments This Year
* I organized, found funding, and continue to drive the new [MLT tile format](https://github.com/maplibre/maplibre-tile-spec) initiative that showed up to 6x size reduction compared with MVT! ([video presentation](https://youtu.be/YHcoAFcsES0?si=9wmZjIkM5PmWIrVK&t=1336)). Special thx to @mactrem!
* I on-boarded and greatly improved [Martin](https://maplibre.org/martin/) - one of the most popular tile server supporting sources like Postgres, MBTiles, PMTiles, Sprites, and Fonts.  On GitHub, 2,100 :star2: and rapidly growing!
* I coordinated the transfer and on-boarded [Maputnik](https://maputnik.github.io/) - map style editor
* I frequently present and do workshops about MapLibre at the conferences. This year: [FOSS4G EU](https://www.youtube.com/watch?v=6y7zA14Jbc8) (+workshop), State of the Map (OSM-US), and plan to attend FOSS4G in Belem.
* I am constantly in communication with community members and sponsors, making sure to hear concerns and community issues.
* I regularly attend monthly sync-ups with the technical steering commeetees (navigation, native, web, and MLT)
* I was actively involved in finding the next coordinator to continue the amazing work done by @wipfli 

### My Mission Statement
I believe MapLibre should offer a complete mapping software stack -- "from bits to pixels": given data, the stack should offer a clear path to convert it into tiles, serve tiles to the world, visualize them with MapLibre web / native / rs, and offer extra functionality with plugins (like navigation), as well as style editing and other tooling. The list goes on.

Yes, but what about next year? I believe MapLibre must have three fundamental goals:
* Grow developer community
* Maintain and improve existing code
* Research next-generation stack with latest technology (WebGPU, faster tiles, multi-platform, ...)

All three goals are important - without the community we won't be able to maintain a vast codebase. Without investment into the next generation technology, we will quickly become irrelevant.

### Financial disclosure
* I have NOT received any payments for my MapLibre work from the organization.
* MapLibre org sometimes reimburses my travel and lodging expenses, but not food or my time.
* I have used some of MapLibre projects in my regular employment

### A little more about me
* Volunteer Wikipedia core developer since 2005, implemented API and bot frameworks. Later worked for the foundation building Maps and data viz.
* Built Sophox OSM Sparql index and other OSM tools and libraries
* Contribute heavily to the Rust ecosystem, including the compiler/libraries
* Former principle engineer at Elastic (ElasticSearch), now working at Rivian (EV cars need good maps!)

Thank you for your support!

**Comment by hy9be** [comment]
My fellow MapLibre community members, 

My name is Haowen You. I’m seeking your support to serve the community as a Governing Board member for the 2025 term. 

As a veteran in the geospatial field, I've been fortunate to witness and contribute to the evolution of geospatial technologies over the past 10 years. My journey began in 2014 when I made several contributions to the first Mapbox Android SDK. In 2016, I co-founded Locus Social, one of the first real-time internet-scale geospatial analytics services. Later while at Grubhub, my team was among the early adopters of [Deck.gl](http://deck.gl/) and GeoSpark. In 2022, I joined AWS as an engineering manager, drawn by the opportunity to grow the MapLibre community and leading Amazon’s contributions to this community.

Since then, I've had the privilege of leading several AWS initiatives to set a strong foundation for MapLibre’s growth. We've provided substantial financial backing, with sponsorships totaling $498K in 2022 and 2023, and we're actively working on renewing our commitment for 2024. I've also guided internal Amazon SDEs in improving documentation and clean up broken dependencies, making the project more accessible to new contributors. Recognizing the community concern in using deprecated rendering API on Apple devices, I assembled a team of domain experts for the Metal migration last year. This year, I have retained everyone on that team and further contributed more than 20 major performance, stability, and maintainability enhancements to the native project. At the same time, I actively engage with Meta, Microsoft, Lyft, Grab, Toyota, and other industry leaders to jointly plan for further adoption and contributions.

Looking ahead, I see the potential for our renderers to become the most stable and performant map renderers available, effectively commoditizing the map rendering technology. My hope is to see MapLibre powering more maps rendered in browsers, phones, vehicles, and on any screens. I'm also excited about the prospect of our new map tile format replacing MVT and becoming the new standard for vector tiles. With all my personal views, however, I firmly believe that MapLibre's future belongs to every community member. As part of the governing board, I see our role as laying the foundation that enables easy adoption, encourages contributions, and inspires and supports innovations from all corners of our community.

If elected to the governing board, my focus for the coming year would be on active listening and community engagement. I commit to speaking with one community member each week, ensuring that diverse voices and perspectives are heard and considered. I aim to connect common interests, facilitate the exchange of ideas, and help navigate diverging interests when they arise. Another significant part of my efforts will be dedicated to driving adoption, increasing contributions, and growing sponsorships to ensure the long-term sustainability and success of MapLibre. 

I'm excited about the possibility of serving on the board and working alongside all of you to shape the future of our community. I would be honored to have your support in this election. 

Thank you.

**Comment by maxammann** [comment]
Hi MapLibre community,

I would love to serve you as a member of the MapLibre board. I believe that I can contribute to the shared vision of MapLibre.

### MapLibre + Me

Since around 2021 I'm following the development of the organization. I have around 13 years of programming, 8 years of professional software development experience, and around 2 years of security engineering experience. Before actively contributing to MapLibre I was invested in finding solutions of open-source offline mapping. I'm a strong believer in the impact and value of FOSS software, and finding a sustainable mapping solution was key to me.

Back then I was a lead developer for the open-source app [Integreat](https://integreat-app.de/en/) which provides local guidance to refugees. The map is an essential feature of the app. Back then, we were directly impacted by the move of Mapbox to become more proprietary. That's why I was very happy when I saw the MapLibre organization was founded. Early on, I was working on the integration of MapLibre into Integreat and other open-source apps.

### Contributions

I contributed to MapLibre the following:

* In 2021 I started the development of maps-rs which was onboarded into the MapLibre organization as maplibre-rs
* One of my first projects after that was to rewrite the maplibre.org website and future-proof it, for which I was paid afterwards.
* In 2022, I met most of the previous board members at conferences like Foss4G where I presented a paper about maplibre-rs or meetups in Switzerland.

### Goals

- First, I want to see what has changed since 2022 and listen to previous board members. I want to do a 1:1 with each of them.
- If feasible, I want to do local meetups in Europe to bring the MapLibre community closer together. I once hosted a meetup for web developers with regular 50+ attendees. 
- Drive forward the R&D aspects of MapLibre. I created maplibre-rs and want to use funding to implement proof-of-concepts for:
  - MLT - the new MapLibre tiles format
  - Advanced text rendering using the Rust ecosystem
  - WebGPU
- Develop and maintain a security concept for MapLibre. That means developing a thread model and pushing initiatives forward that improve MapLibre's security posture.

I recently quit my job in IT security, so I have plenty of time to move forward with these goals.

### What else?

You may have noticed that I have not mentioned the current maplibre-native and maplibre-gl-js projects so far.
I'm still maintaining several apps using maplibre-native so I'm committed to keeping these projects well-maintained and bringing planned features to them.
These libraries are the core of MapLibre and those are the projects that bring funding from large organizations. They are essential to them.
However, I want to be open and honest that my focus is the R&D aspects of MapLibre. I'm interested in bringing technology that is state-of-the-art or even above that to MapLibre.

**Comment by birkskyum** [comment]
Dear community, 

I'm a co-founder of maplibre, and I've been on the board since the first election cycle, and it's been some constructive years. I've been honored to represent the community for this long, and I'd love to continue my work for another term.

There is a lot going on right now on all fronts, which is awesome. To give an idea about some of the concrete work I've been leading in the past year, here's a review.

# Efforts in the Past year

From 2021-2023 I spent the majority of my time working on tooling, general maintenance, testing, modern node compat, CI for native and gl js, typescript, splitting out style spec, docs etc., but that all runs well now, and we even have excellent maintainers that keeps improving on it.

This past year I directed my attention more towards community and R&D:

### DevRel - Community and ecosystem
 I've been focusing on helping onboard projects that can benefit from MapLibre to our community, and make sure our compatibility in the ecosystem stays high. Notable examples are:

- [Plotly.js](https://github.com/plotly/plotly.js) (MIT-license), an awesome python plotting library, with 4m weekly downloads, that will have support maplibre from the next minor (RC release any day now), and exclusively support maplibre from next major. ( https://github.com/plotly/plotly.js/pull/7015#issuecomment-2275965617 )

- [Kepler.gl](https://kepler.gl/) (MIT-license), a great tool for data visualization, which now runs on maplibre instead of mapbox, so the entire stack is foss again. ( https://github.com/keplergl/kepler.gl/pull/2461 )

- [Deck.gl](https://deck.gl/) (MIT-license), I've been trying to facilitate better sync with deck.gl - it was requested maplibre exposed more camera internals, which is in place as part of MapLibre GL JS 4.5.2. ( https://github.com/maplibre/maplibre-gl-js/pull/3136 )

- [OpenPilot](https://github.com/commaai/openpilot) (MIT-license) - Took contact to the team. It's running MapLibre Native now https://github.com/commaai/openpilot/issues/26739

I expect to start collaborating with this team soon - there's a vote on this change in progress.
- [Apache Superset](https://github.com/apache/superset) (ofc. Apache license - 61k GitHub stars) - https://github.com/apache/superset/issues/28356

### Breaking new ground - Healthy competition amongst our renderers
In the past year I've been thinking a lot about our long-term strategy, around if we can unify maplibre to a single renderer. In that case we have to figure if it's possible to make MapLibre Native run on the web, and if MapLibre GL JS can be fast enough for mobile use, or if it's better to introduce a third renderer that's a bit of both. I don't know the answer here, but I know that we can pursue each to learn more, and then reassess at a later point. My work in this regard is:

_**Can we make Native portable?**_

I've been doing some pioneering work regarding exploring our options for [MapLibre Native Wasm](https://birkskyum.github.io/maplibre-native-wasm/) compilation to run in the browser, which I demonstrated with our first Wasm demo [here](https://birkskyum-maplibre-native-wasm.pages.dev/qt-for-webassembly/webgl1-from-opengl2-legacy-renderer/). This could be the first baby step towards increasing the portability of Native to web, thus making it truly cross-platform.

_**Can we make GL JS fast?**_ 

I push for our MapLibre GL JS to move forward towards a [modular graphics backend](https://github.com/maplibre/maplibre-gl-js/issues/4511), so that we can begin refactor to adopt faster WebGL2 and WebGPU functionality. I started [migrating to webgl2](https://github.com/maplibre/maplibre-gl-js/pull/2599) a year ago, but due to a lack of this refactor we had to put this work on hold, and even down-compile to webgl1. If we can make GL JS run super fast, and it already have super high portability, and a much simpler codebase, then I believe it can stand the pressure of the Native renderers.

Thanks,
Birk

**Comment by lseelenbinder** [comment]
Hi, I'm Luke.

I've been part of MapLibre since the beginning—helping with the initial communication, organizational setup, and a member of the governing board since inception. I hope to continue serving the community as a member of the board and active participant. This year continued that work, and I had the chance to speak several times about MapLibre, and see the completion of several of my goals from last year.

This past year saw the conclusion of the majority of the Metal transition work, moves towards improving and overhauling the bounty system in favor of more in-house talent that can directly build the libraries, and some initial work towards unified code in Rust (thanks @nyurik!).

For the upcoming year, I hope to see the following achieved:

- A strong organization with the new graphics engineer and new coordinator.
- MapLibre Tiles succeed in moving beyond the POC to a viable production project.
- Continued growth and success of the core projects and their communities.
- Continued support from our generous sponsors and community members.

Personally, I'm also invested in seeing an adjacent navigation project, spearheaded by my company, be more closely integrated with the MapLibre ecosystem so there's once again a production-ready, open source navigation SDK for all to use.

Here's to another year of building tools to help everyone build the best open source maps!

**Comment by klokan** [comment]
Hey, everyone

my name is [Petr Pridal](https://maplibre.org/about/petr/), known as Klokan. I am co-founder of MapLibre, founder of OpenMapTiles, EPSG.io, author of GDAL2Tiles, architect of OldMapsOnline.org, contributor to open-source cartography on multiple fronts and projects, entrepreneur and CEO of MapTiler.com.

I am keen to offer my services to the MapLibre project and community again - as self-nominee for the governing board in 2025, because I strongly believe in the viability of open-source components for developers.

In the upcoming year, I want to help to push forward:
- [MapLibre Globe visualization](https://labs.maptiler.com/showcase/globe/) as production ready functionality for everyone (huge thanks @windycom @kubapelc already!)
[![maplibre-globe](https://github.com/user-attachments/assets/896961e5-a3fe-4a06-8e85-b3a981450c49)](https://labs.maptiler.com/showcase/globe/)

- Kick-off new [WebGPU](https://en.wikipedia.org/wiki/WebGPU) renderer in TypeScript - to benefit from GPU accelerated processing of label collisions, data parsing & handling, transformations, improved font rendering, well-structured web workers, and prepare the map library for the integration of accelerated AI inference in the future. Let's gain more computational boost on modern hardware, which is closer to Vulkan and Metal instead of decades old OpenGL - while working on web-first code, but keep in sync with mobile and backend platforms
- Ensure amazing benefits of growing [MLT - MapLibre Tile format]( https://github.com/maplibre/maplibre-tile-spec) becomes reality in production, but does not slow down stable GL JS - with the biggest optimizations to be made in the new WebGPU renderer
- Support further [performance boosts](https://labs.maptiler.com/sdk-benchmarks/) for MapLibre to build the fastest rendering map library out there

Please vote for me, so I can continue to contribute with improvement of processes and governance of the project - while helping to shape the future with strategical insights from industry and with my executive experience.

I have a degree in Computer Science, I have been coding myself for years, I also have extensive practical experience in management as well as education in board governance from IMD in Switzerland.

In the past, I was fighting on the board of MapLibre for:

- clear processes - hiring + documented roles & responsibilities (see subpages at [wiki](https://github.com/maplibre/maplibre/wiki))
- measurable goals aligned to [the project strategy](https://drive.google.com/file/d/1F5abpGx8VlVse4SHGwxgRRY1wQ29i-0_/view)
- transparent accounting (have a look yourself at https://opencollective.com/maplibre#category-BUDGET)
- clear [sponsorship offer](https://maplibre.org/sponsors/Slide-Deck-MapLibre-Sponsorship-Program.pdf) leading to raising money, allowing employment of maintainers in this open-source project
- trademarks - I donated the MapLibre EU trademark to the organization
- focus on becoming "the best rendering library" out there with innovation and clear roadmap

In previous years me and my team helped with setting up of this project, branding and identity of the project, we contributed in code on the initial version of Native, pushed initial announcements and assisted on many other fronts.

Let’s bring the MapLibre project to the next level - together :-)
